src/client/sandbox/native-methods.js

```diff
@@ -199,7 +199,8 @@ class NativeMethods {
         this.elementChildElementCountGetter = win.Object.getOwnPropertyDescriptor(window.Element.prototype, 'childElementCount').get;
 
         if (win.PerformanceEntry)
-            this.performanceEntryNameGetter = win.Object.getOwnPropertyDescriptor(window.PerformanceEntry.prototype, 'name').get;
+            if (win.PerformanceNavigationTiming)
```

Comment:
I guess the only one condition `if (win.PerformanceNavigationTiming)` is enough.

Reply:
Thanks, I found it out in documentation: "PerformanceEntry instances will always be one of the following subtypes: ..., PerformanceNavigationTiming, ... "